src/lib/pages/backoffice/Acquisition/Order/OrderDetails/OrderStatistics.js

```diff
@@ -91,10 +91,14 @@ export class OrderStatistics extends React.Component {
     return (
       <Statistic>
         <Statistic.Label>Total</Statistic.Label>
-        <Statistic.Value>
-          {formatPrice(order.grand_total, false)}
-        </Statistic.Value>
-        <Statistic.Label>{order.grand_total.currency}</Statistic.Label>
+        {order.grand_total ? (
```

Comment:
If I am not wrong this will display a label `Total` without any element in it, would it make sense to render the `<Statistic>` element only if there is a grand.total_value? Another approach would be to display the `<Statistic>` element and if it doesn't have a value in grand_total to display a `-` or in statistic value and currency, this will also work as expected with the styles

Reply:
Hmm, I kept it like this because it's harmonized with the rest of the page, if we remove the `<Statistic>` component then nothing will appear at the top right of the image. 
I agree with you to add a default value for empty fields but I think we should update the rest of the fields too to prevent different "styles" in the same page.
![image](https://user-images.githubusercontent.com/15194802/99259893-f800e100-281a-11eb-8561-4b2da3ccfe5b.png)